stories/base/Pagination.stories.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from "react";
+import styled from "@emotion/styled";
+import { storiesOf } from "@storybook/react";
+import Paginaton from "../../components/base/Pagination";
+import { maxWidth, BREAKPOINT_MOBILE } from "../../components/breakpoints";
+
+// Wrapper div to positiont the pagination links
+const PaginationWrapper = styled.div`
+  padding: 50px;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  &.center {
+    top: 0;
+    justify-content: center;
+  }
+  /* Previous and Next boxes are stacked on a smaller screen */
+  @media (${maxWidth(BREAKPOINT_MOBILE)}) {
+    flex-direction: column;
+    a {
+      margin: 10px 0;
+    }
+  }
+`;
+
+storiesOf("base|Pagination", module)
+  .add("Next", () => (
+    <PaginationWrapper className="center">
+      <Paginaton variant="next" variantText="NEXT" href="#">
```

Comment:
This is probably a nitpick, but can we just use `text-transform: uppercase` on the variant text? I feel like that leaves less room for inconsistencies/typos while maintaining this down the road

Reply:
That's good point! 👍

Reply:
Should be address by this [commit](https://github.com/Dwolla/dev-portal/pull/38/commits/2cf01f7d6d3eded6cbcd2dbe20faa810c278d2fa)!